samples/shields/x_nucleo_iks01a1/prj.conf

```diff
@@ -1,2 +1 @@
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_SHIELD_X_NUCLEO_IKS01A1=y
```

Comment:
Besides providing a way to select the matching overlay this config symbol was used to do the shield configuration in Kconfig.defconfig and Kconfig.shield.
So, we have 2 options:
- Benefit in application from provided configuration provided by shield. So keep this line.
- Argue that board + shield = new board. New board should then adhere to default configuration guidelines and should let features activation up to application, so remove Kconfig.defconfig and Kconfig.shield and add equivalent configuration in place of shield flag:
```
CONFIG_I2C=y
CONFIG_I2C_STM32_INTERRUPT=y
CONFIG_SENSOR=y
CONFIG_HTS221=y
CONFIG_LPS25HB=y
CONFIG_LSM6DS0=y
CONFIG_LIS3MDL=y
CONFIG_HTS221_TRIGGER_NONE=y
CONFIG_LIS3MDL_TRIGGER_NONE=y
```
I find the second approach more coherent.